### PR TITLE
Content Audit Tool is now in BETA

### PR DESCRIPTION
**Pull request by andrewgarner**

### CODE REVIEW

**Comment by andrewgarner** [comment]
[Add feedback link to Beta banner](https://trello.com/c/GZAcO0DK/542-add-feedback-link-to-beta-banner)

**Comment by alecgibson** [comment]
Given that this is one huge `if` block, wouldn't it better to just have two separate Alpha and Beta templates? Things would be a lot more readable that way.

File: app/views/layouts/audits.html.erb [hunk 1]
```diff
@@ -17,7 +17,7 @@
 <% end %>
 
 <% content_for :content do %>
-  <%= render 'application/phase_banner' %>
+  <%= render 'application/phase_banner', phase: :beta %>
```
Comment (pmanrubia) [inline app/views/layouts/audits.html.erb]: Have you thought in splitting `phase_banner` into two different files?. We are passing a variable and then [checking it inside the partial with no big benefit](https://github.com/alphagov/content-performance-manager/pull/352/files#diff-211f3c6952675e67d0f1f1bdf4f9c8dcR7).

In the layout, we know what phase we are in, so we should be able to pick up the right one.

**Comment by andrewgarner** [comment]
@pmanrubia @alecgibson – changes made.

**Pull request merged**